Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,50 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+operational direction of the Node.js Technical Steering Committee (TSC).
+
+## Organization Roles
+
+Individual users within a GitHub organization may be granted one of three
+distinct roles: Member, Owner, or Billing manager. The permissions granted
+to each role are documented in the GitHub documentated [Permission levels for
+an organization](https://help.github.com/articles/permission-levels-for-an-organization/).
+
+### Billing manager
+
+All Billing managers for the Node.js Foundation GitHub Organization shall
+be determined by the Node.js Foundation Board of Directors with notification
+given to the TSC.
+
+### Owners
+
+All voting members of the Node.js TSC shall have Owner permissions within the
+Node.js Foundation GitHub Organization. Additional users may be granted Owner
+permissions at the discretion of the TSC through normal TSC motion and simple
+majority vote.
+
+### Members
+
+GitHub users may be granted Member status in the Node.js Foundation GitHub
+Organization by any Owner.
```

Comment:
First of all, all of the comments are out of line with the existing processes.

Second, we allow individual WGs to define their own criteria and process for adding members (which in-turn become org members). What we **should not do** is define a process here that restricts their ability to continue to define and add members at their own discretion.

If the document is to remain purely administrative it would say something along the lines of "GitHub users are added to the org when they are added to a Working Group. Org Owners should add new members to orgs/teams when requested by a WG."